Get-NetView.psm1

```diff
@@ -2306,8 +2297,8 @@ function Completion {
     which can be accessed by using "$CustomModule" as a placeholder. For example, {Copy-Item .\MyFile.txt $CustomModule}
     copies "MyFile.txt" to "CustomModule\MyFile.txt".
 
-.PARAMETER MaxThreads
-    Maximum number of simultaneous background tasks, from 1 to 16. Defaults to 5.
+.PARAMETER BackgroundThreads
+    Maximum number of background tasks, from none up to 16. Defaults to 5.
```

Comment:
Change "from none" to "from 0 - N".  Where N should be the upper bound.  

Why cap the max threads at 16?  IMHO, capping at 16 is somewhat artificial.  Why not allow the user to set 64 or 128 if they wish?

Reply:
I actually see slightly worse performance with high threads count, with maximum benefit achieved at about 8 threads. This is a system with 20 cores / 40 LPs, so it isn't clear what the bottleneck is, or how other systems will scale.

Reply:
Given your performance results, should the default be 8 then?  Instead of 5.